demo/lib/vm.bash

```diff
@@ -379,10 +389,11 @@ vm-install-cri-resmgr() {
         suf=$(vm-pkg-type)
         vm-command "rm -f *.$suf"
         local pkg_count
-        pkg_count=$(ls "$HOST_PROJECT_DIR/$binsrc"/cri-resource-manager*.$suf | grep -v dbg | wc -l)
+        # shellcheck disable=SC2010,SC2126
+        pkg_count=$(ls "$HOST_PROJECT_DIR/$binsrc"/cri-resource-manager*."$suf" | grep -v dbg | wc -l)
```

Comment:
nitpick: again I don't understand why shellcheck does not warn here about $() lacking enclosing quotes. Seems pretty bogus to me.

Reply:
Double quotes added.